docs/source/pages/tutorials/commit_finder.rst

```diff
@@ -5,39 +5,47 @@
 Analyzing and comparing different versions of an artifact
 ---------------------------------------------------------
 
-This tutorial demonstrates how Macaron can be used to determine the differences between one or more states of the single open-source repository that produced one or more related artifacts. In this way, we show how a developer can be potentially misled by supply chain security information that has been created for the current state of an artifact's source repository, rather than the version of the artifact they are actually using.
+This tutorial demonstrates how Macaron can be used to determine the differences between one or more
+states of the single open-source repository that produced one or more related artifacts. In this way,
+we show how a developer can be potentially misled by supply chain security information that has been
+created for the current state of an artifact's source repository, rather than the version of the
+artifact they are actually using.
 
-he problem of mapping artifacts to the source code that built them is a challenging one, as most artifacts, even open-source ones, do not provide a direct URL to the related repository and commit. In principle, provenances, such as :term:`SLSA` or :term:`Witness`, which contain the commit that was used to build the artifact provide the information that we need. However, currently the adoption rate in the open-source community is low, therefore limiting its value for this task.
+It is challenging to map artifacts to the source code because most artifacts, even open-source ones, do
+not provide a direct URL to the related repository and commit:
```

Comment:
Same here for 14 to 15.

Reply:
See commit [441888a](https://github.com/oracle/macaron/pull/606/commits/441888afdb2cd23b20be86fe1e55ec0be23a2af1)